superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

```diff
@@ -533,13 +539,7 @@ function DashboardList(props: DashboardListProps) {
           { label: t('No'), value: false },
         ],
       },
-      {
-        Header: t('Search'),
         key: 'search',
```

Comment:
Oops! I think this line needed to come along for the ride!

Reply:
Ah apologies, forgot to check CI